routes/index.js

```diff
@@ -2,7 +2,8 @@ const express = require("express");
 const router = express.Router();
 
 const lectureRouter = require("./lectureRouter");
-
+const userRouter = require("./userRouter")
```

Comment:
가독성을 위해 아래 공백을 한 줄 추가해 주면 좋을 것 같습니다.
```suggestion
const userRouter = require("./userRouter")

```